DOCKER.md

```diff
@@ -148,3 +149,26 @@ docker run -v $(pwd)/config.json:/usr/src/app/config.json -p 8000:8000 -d scalit
 
 Your local `config.json` file will override the default one through a docker
 file mapping.
+
+### Running as an unprivileged user
+
+S3 Server runs as root by default.
+
+You can change that by modifing the dockerfile and specifying a user before the entrypoint.
+
+The user needs to exist within the container,
+and own the folder **/usr/src/app** for Scality S3 Server to run properly.
+
+For instance, you can modify these lines in the dockerfile:
+
+```shell
+...
+&& groupadd -r -g 1666 scality \
```

Comment:
I could be wrong, but I think that the node user already use this number.

I can use any number you like.